repositories.yml

```diff
@@ -3,49 +3,173 @@ section:
+          or init container to ensure your application has a valid Conjur access token.
+        certification: "certified"
+        version: v0.15.0
+      - name: cyberark/secrets-provider-for-k8s
+        url: https://github.com/cyberark/secrets-provider-for-k8s
+        tool: Kubernetes
+        description: The Conjur Secrets Provider for K8s is deployed as an init
+          container in your application pod. It injects secrets from Conjur
+          into Kubernetes secrets, which are accessible to your application pod.
+        certification: "certified"
+        version: v0.2.0
+      - name: cyberark/conjur-service-broker
+        url: https://github.com/cyberark/conjur-service-broker
+        tool: Cloud Foundry
+        description: The Conjur service broker ensures your Cloud Foundry-deployed
+          applications are bootstrapped with a Conjur machine identity on deploy.
+        certification: "certified"
+        version: v1.0.0
+      - name: cyberark/cloudfoundry-conjur-buildpack
+        url: https://github.com/cyberark/cloudfoundry-conjur-buildpack
+        tool: Cloud Foundry
+        description: The Conjur buildpack brings the benefit of Summon to
+          your Cloud Foundry-deployed applications. Leverage your app's Conjur identity
+          to automatically inject the secrets your app needs into its environment
+          at runtime.
+        certification: "certified"
+        version: v2.1.1
+
+  - name: DevOps Tools
+    description: Conjur OSS integrations with DevOps tools.
+    repos:
+      - name: cyberark/ansible-conjur-host-identity
+        url: https://github.com/cyberark/ansible-conjur-host-identity
+        tool: Ansible
+        description: Ansible role to provide Conjur machine identity to application
+          hosts and install the Summon tool, which enables hosts to securely retrieve
+          credentials.
+        certification: "trusted"
+        version: v0.3.0
+      - name: cyberark/conjur-credentials-plugin
+        url: https://github.com/cyberark/conjur-credentials-plugin
+        tool: Jenkins
+        description: Conjur plugin for securely providing credentials to Jenkins jobs.
+        certification: "community"
+        version: v0.7
+      - name: cyberark/conjur-puppet
+        url: https://github.com/cyberark/conjur-puppet
+        tool: Puppet
+        description: Puppet module that simplifies the operation of establishing
+          Conjur host identity and allows authorized Puppet nodes to fetch secrets
+          from Conjur.
+        certification: "trusted"
+        version: v2.0.1
+      - name: cyberark/terraform-provider-conjur
+        url: https://github.com/cyberark/terraform-provider-conjur
+        tool: Terraform
+        description: Terraform provider that makes secrets in Conjur available in
+          Terraform manifests.
+        certification: "certified"
+        version: v0.1.0
+        
+  - name: Secretless Broker
```

Comment:
Broker and summon sections should probably go before platform integration category

Reply:
why?

Reply:
they're building blocks of conjur ecosystem vs the integration that pulls them together so if you're doing the listing of categories by importance, integrations/devtools should be last and the others should be above. You can also do the list alphabetically if you want but I'm guessing that you don't want that ordering in resulting output.

Reply:
I'm not sure I see the distinction between the "integrations" that enable you to use conjur with your devops tools and platforms and the "delivery mechanisms" that ensure your apps can get secrets / access to services that they need

I guess I have them ordered this way because in terms of the flow of deploying things in my ecosystem, I need to deploy conjur first (and the api-related tools belong with conjur), and then I use the integrations to get my apps deployed, and with my apps deployed they invoke summon or secretless. does that make some kind of sense?

Reply:
deployment order still seems backwards I guess:
```
then I use the integrations to get my apps deployed
```
broker and summon are usually part of that "app"

Reply:
but they are not invoked until the app is deployed

honestly, it's not critical. but in my head they're called "last mile secrets delivery" and so I put them last

Reply:
I'll mark this thread resolved since it's not important enough to block the merge